architecture/src/commonTest/kotlin/observable/ReadWritePropertyTest.kt

```diff
@@ -57,7 +57,10 @@ class ReadWritePropertyTest : ObservableBaseTest() {
 
         nullableReadWritePropertyValue.value = initialValue
 
-        val subject = nullableReadWriteProperty.toDefaultSubject("default", Dispatchers.Unconfined)
```

Comment:
please also add tests for the cases 
1. `coroutineScope` is null 
2. `coroutineScope` is not null, but context is set to null explicitly